docs/README.md

```diff
@@ -1,81 +1,96 @@
-|**-i:** |Path to ignored packages file.| YES|_-i:c:\files\ignore.txt_|
-|**-idut** |Indicates that packages relevant to the .NET Dev UX team are ignored. If **-i:** is also set, the file specified with that option is used, superceding **-idut**.|YES|> _-idut_|
-|**-wl:** |Path to the whitelist file [**optional**]. If this is specified, only the packages listed in this file will be updated. Each line in the file represents a regex pattern that will potentially match one or more package IDs. This switch will be ignored if the given file is empty.|YES|> _-wl:c:\files\whitelist.txt_|
-|**-p**: |Path to the directory to search for .props files. If left unspecified, all the .props files under src\SetupPackages in local VS repo will be searched.|YES|`_-p:C:\VS\src\SetupPackages_`|
-|**-a**: |Personal access token to access packages in [VS feed](https://pkgs.dev.azure.com/devdiv/_packaging/VS-CoreXtFeeds/nuget/v3/index.json). If not specified, props files will not be updated.|YES|`_-a:vv8ofhtojf7xuhehrFxq9k5zvvxstrqg2dzsedhlu757_`|
-|**-w:** |Maximum allowed duration in seconds, excluding downloads.|YES  |`> _-w:60_`|
-|**-ds:** |Maximum allowed duration in seconds that can be spent downloading nuget packages |YES|`> _-ds:240_`|
-|**-c:** |Maximum concurrency of default.config version updates.|YES|`> _-c:10_`|
-|**-bf:** |Filter string to specify which builds from the manifest will be inserted.| YES|A simple build filter can be written as follows:</br>`-bf:repo=.*core-setup`|
+|**-d:** |Path to the default.config file.| :x: | `-d:c:\default.config`|
+|**-m:** |Path to the manifest.json file or to the containing directory.| :x: | `-m:c:\files\manifest.json`<br/>or<br/> `-m:c:\files` |
+|**-i:** |Path to ignored packages file.| :heavy_check_mark: | `-i:c:\files\ignore.txt` |
+|**-idut** |Indicates that the packages relevant to the .NET Dev UX team are ignored. If both set, **-i:** option overrides this switch.|:heavy_check_mark:| `-idut`|
+|**-wl:** |Path to the allowlist file. If this is specified, only the packages listed in this file will be updated. Each line in the file represents a regex pattern that will potentially match one or more package IDs. This switch will be ignored if the given file is empty.| :heavy_check_mark: | `-wl:c:\files\allowlist.txt` |
+|**-p**: |Path to the directory to search for .props files. If left unspecified, all the .props files under src\SetupPackages in the local VS repo will be searched.| :heavy_check_mark: |`-p:C:\VS\src\SetupPackages`|
+|**-a**: |Personal access token with "read" access to the packages in the [VS feed](https://pkgs.dev.azure.com/devdiv/_packaging/VS-CoreXtFeeds/nuget/v3/index.json). If not specified, props files will not be updated.| :heavy_check_mark: | `-a:vv8ofhtojf7xuhehrFxq9k5zvvxstrqg2dzsedhlu757` |

```

Comment:
nit: The token is quite long and it is unnecessary information itself. I wonder, what if we shorten it a bit, remove a couple of letters. Likely nobody will notice but the line will fit better. Or just put `...` in the middle.